script.copacetic.helper/plugin.py

```diff
@@ -0,0 +1,6 @@
+# author: realcopacetic, sualfred
+
+from resources.lib.plugin.main import Main
+
+if __name__ == '__main__':
+    Main()
```

Comment:
dummy init python files are no longer needed in python3

Reply:
Thanks @enen92my coding knowledge is fairly limited. Can you point me towards an example of how this should be done in python 3?

Reply:
Thanks @enen92 my coding knowledge is fairly limited. Can you point me towards an example of how this should be done in python 3?

Reply:
Just remove the empty `__init__.py` that is below in the GitHub diff

Reply:
I'd rather disagree. While Python 3 supports so called implicit namespace packages, for regular cases it's better to use old style packages with `__init__.py`.